Module/uti/cartelkeycard.uti.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
As mentioned on Discord - I don't see this key card referenced in the above modified area files. Where is this used?